plugins/modules/database/postgresql/postgresql_privs.py

```diff
@@ -561,10 +562,16 @@ def get_all_sequences_in_schema(self, schema):
     def get_all_functions_in_schema(self, schema):
         if not self.schema_exists(schema):
             raise Error('Schema "%s" does not exist.' % schema)
-        query = """SELECT p.proname, oidvectortypes(p.proargtypes)
-                    FROM pg_catalog.pg_proc p
-                    JOIN pg_namespace n ON n.oid = p.pronamespace
-                    WHERE nspname = %s"""
+        if self.pg_version >= 110000:
+            query = ("SELECT p.proname, oidvectortypes(p.proargtypes) "
+                     "FROM pg_catalog.pg_proc p "
+                     "JOIN pg_namespace n ON n.oid = p.pronamespace "
+                     "WHERE nspname = %s and p.prokind = 'f'")
```

Comment:
The only difference between the two queries is the last condition here, right? Maybe instead duplicating the query, it would be better to create a default query and append the additional condition in an if block (`if self.pg_version < 110000`)?

Reply:
(I thought I left the comment but seems it was cancelled by mistake)
Yep, thanks, now it looks better!